### ISSUE TITLE
Add basic styling for the [charitable_my_donations] table

### ISSUE THREAD

**Issue by ericnicolaas** [Charitable/Charitable#367]
Right now this is very barebones — just markup, no styling.

![screenshot 2017-08-08 09 52 21](https://user-images.githubusercontent.com/676582/29051082-48cb8cd0-7c1f-11e7-948b-f6bade3e0a90.png)

A second question is whether to keep the table based markup, or switch to a div or li for each receipt. The table is simple, but harder to style and not ideal for responsive purposes. Switching to div/li instead could give us a bit more flexibility in how we style these.

**Comment by ericnicolaas** [comment]
@wesfranklin Can you mock something up for this? Doesn't need to be the table layout like above; I'm thinking maybe a larger block for each receipt, keeping in mind some spots where we can introduce flexibility (i.e. spots to add more links/information).

**Comment by wesfranklin** [comment]
@ericnicolaas What sort of additional info/links would you want to add?

**Comment by wesfranklin** [comment]
@ericnicolaas Not quite sure how to do this. You could have extra columns for 'More' and 'Receipt', but what do you think about this option?
![screen shot 2017-08-08 at 11 22 38 am](https://user-images.githubusercontent.com/13580400/29052901-0a1f1bf2-7c2c-11e7-8a52-c85b0117cc7b.png)

**Comment by wesfranklin** [comment]
Give styling: https://givewp.com/addons/pdf-receipts/

**Comment by ericnicolaas** [comment]
@wesfranklin Yeah, that's quite a nice option, and wouldn't be too far removed from what we have now.

What do you think about making it a bit more engaging by including a photo for the campaign that was donated to? See below a screenshot from one of the organisations that we make a regular donation to:

![compassion_screenshot](https://user-images.githubusercontent.com/676582/29255384-feb4943c-80df-11e7-8ec2-da6d61ea0014.png)

**Comment by wesfranklin** [comment]
@ericnicolaas Yep, that'd be good.

**Comment by helgatheviking** [comment]
Would love this. Tables looks pretty rough in Recurring Donations too:
![image](https://user-images.githubusercontent.com/507025/33686537-d8ffaffa-da9a-11e7-82df-1ad581a09220.png)

**Comment by wesfranklin** [comment]
@helgatheviking For now, I think just a basic table to make the columns clearer would be good. Not sure if @ericnicolaas has done much on this for the My Donations page?

**Comment by helgatheviking** [comment]
I'm cool with that. My screenshot IS the my donations page. The top half is the charitable donations shortcode. I'm going to add the "my recurring donations" right after that automatically, but also make it available as its own shortcode. I don't think there's any styling in Charitable yet for this, but would definitely add any classes required to get the styling essentially for free in Recurring.

**Comment by ericnicolaas** [comment]
For the sake of improving this in 1.6, I have just added a slight bit of padding to the right of the cells. In 1.7 we can explore doing something a little more adventurous here.
